assets/js/base/components/pagination/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Label from '../label';
+import { getIndexes } from './utils.js';
+import './style.scss';
+
+const Pagination = ( { currentPage, pagesToDisplay, onPageChange, totalPages } ) => {
+	const { minIndex, maxIndex } = getIndexes( pagesToDisplay, currentPage, totalPages );
+	const pages = [];
+	for ( let i = minIndex; i <= maxIndex; i++ ) {
+		pages.push( i );
+	}
+	const showPreviousArrow = minIndex !== 1;
```

Comment:
Is this logic correct? 

Previous arrow showing on non page 1 is correct, but the arrow itself jumps to page 1, not the previous page. It's also labelled as "first page".

Reply:
Discussed here: https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/926#discussion_r319920033.